app/src/main/java/com/bashkevich/androidfundamentals/model/EntityMapper.kt

```diff
@@ -0,0 +1,32 @@
+package com.bashkevich.androidfundamentals.model
+
+import com.bashkevich.androidfundamentals.model.dto.ActorDto
+import com.bashkevich.androidfundamentals.model.dto.GenreDto
+import com.bashkevich.androidfundamentals.model.dto.MovieDto
+import com.bashkevich.androidfundamentals.model.entity.Actor
+import com.bashkevich.androidfundamentals.model.entity.Genre
+import com.bashkevich.androidfundamentals.model.entity.Movie
+
+object EntityMapper {
+    fun convertToMovieFromDto(movieDto: MovieDto): Movie {
+
+        return Movie(
+            movieDto.id,
+            movieDto.title,
+            movieDto.overview,
+            movieDto.posterPicture?.let { it },
+            movieDto.backdropPicture?.let { it },
+            movieDto.ratings,
+            movieDto.votesCount,
+            if (movieDto.adult) 16 else 13,
+            movieDto.runtime?.let { it },
```

Comment:
Same

Reply:
when I get a list of movies(e. g. top_rated)? there is no runtime

Reply:
That is file, just make it null by default if need